js_task2/src/js/eventFunctions.js

```diff
@@ -0,0 +1,48 @@
+import { checkedItems, baseUrl, users, numOfRows } from './_variables';
+import { deleteUserNote } from './dataFunctions';
+import { setDataToObject } from './helpers';
+import ModalWindow from './ModalWindow';
+
+export const addRowSelectionEvent = (row) => {
+  row.addEventListener('click', (Event) => {
+    if (Event.target.type === 'button') {
+      return;
+    }
+
+    let checkbox = row.querySelector('input[type="checkbox');
```

Comment:
re-check please this selector, it should be 'input[type="checkbox"]'

Reply:
thx, changed